src/test/java/org/folio/rest/api/CirculationRulesApiTest.java

```diff
@@ -72,7 +72,7 @@ private JsonResponse putResponse(CirculationRules circulationRules) throws Excep
   /** @return the message of the put response, asserts an HTTP_OK=200 response */
   private String put204(CirculationRules circulationRules) throws Exception {
     JsonResponse response = putResponse(circulationRules);
-    assertThat(response.getBody(), response.getStatusCode(), is(HttpURLConnection.HTTP_NO_CONTENT));
+    assertThat(response.getBody(), response, matchesNoContent());
```

Comment:
```suggestion
    assertThat(response, matchesNoContent());
```
matchesSafely automatically appends response.getBody() to the failure message.